ui/src/me/components/account/TokenRow.tsx

```diff
@@ -87,6 +87,12 @@ class TokenRow extends PureComponent<Props> {
     const {onClickDescription, auth} = this.props
     onClickDescription(auth.id)
   }
+
+  private handleUpdateName = async (value: string) => {
+    const {auth, onUpdate} = this.props
+    auth.description = value
+    await onUpdate(auth)
```

Comment:
might be better to use a spread here `{...auth, description: value}` rather than mutating the object. If you change the name from value to description you can even shorten it a bit.